.gitignore

```diff
@@ -1 +1,2 @@
 *.kdev4
+build
```

Comment:
I usually use out-of-source builds (i.e. I make the build directory a sibling not a child of the project directory), but why not.